compiler/rustc_passes/src/stability.rs

```diff
@@ -791,10 +814,17 @@ impl<'tcx> Visitor<'tcx> for Checker<'tcx> {
                                 &self.tcx.sess.parse_sess,
```

Comment:
Given https://github.com/rust-lang/rust/issues/55149#issuecomment-1159521351 has completed, maybe you would be up for dropping the feature gate and related code in this PR?